tasks/bot-avatars.md

```diff
@@ -131,7 +131,7 @@ on things from this list.  Be creative!
 - sales bot (bot that plugs into your Etsy/Squarespace/Shopfiy/Stripe/Square/etc
```

Comment:
As an example: here, "bot that plugs" is correct. :)

But, as long as you're fixing things, please fix the spelling of "Shopify" in this line!